product_docs/docs/efm/5/efm_rel_notes/29_efm_50_rel_notes.mdx

```diff
@@ -0,0 +1,33 @@
+
+| Type | Description |
+| ---- |------------ |
+| Enhancement | The startup log is no longer used by Failover Manager agents. Agent startup information now goes to systemd or standard out, depending on how the agent was started. |
+| Enhancement | New CLI command 'efm create-standby' added to create standby databases from an existing primary using 'pg_basebackup'. |
+| Enhancement | Failover Manager now simplifies handling cluster size for quorum purposes. Failed nodes/agents are still part of a cluster until removed. |
+| Enhancement | Added new 'backup.wal' property to control whether or not standby WAL is backed up during reconfiguration. [Support ticket: #1277862, #91506, #101026] |
+| Enhancement | The 'auto.resume.period' property has been split into two new properties, one for agent startup and one for database failures. |
+| Enhancement | Added new 'check.num.sync.period' property to control how often a primary database is checked for synchronous standbys. [Support ticket: #1133047, #91506] |
+| Enhancement | Removed unnecessary check of database operating system user in sudoers conf file. [Support ticket: #42594, #99957] |
+| Enhancement | Removed unnecessary network status check that was added for older operating systems. |
+| Enhancement | Table information in 'efm cluster-status' output will now resize to handle long host names. [Support ticket: #101878] |
+| Enhancement | Updated xlog references in 'efm cluster-status-json' output to use "lsn" instead. |
+| Enhancement | Changed the default 'detach.on.agent.failure' behavior to match the suggested value. |
+| Enhancement | Added more cluster size checks during failures to improve split brain safety during network outages that result in gradual node isolation. |
+| Enhancement | Removed local database checks after an agent cannot connect that produced confusing logging. |
+| Enhancement | Added more information in some cases when an agent cannot connect to a database. |
+| Enhancement | Failover Manager was upgraded to use JGroups version 5.4.4. |
+| Enhancement | Failover Manager was upgraded to use PostgreSQL JDBC Driver version 42.7.3. |
+| Enhancement | Failover Manager was upgraded to use Log4J version 2.24.3. |
+| Enhancement | Failover Manager was upgraded to use Jakarta Mail version 2.0.1. |
+| Enhancement | Failover Manager was upgraded to use Json-Simple version 1.1.1. |
+| Bug Fix | Fixed agent shutdown code that could cause a 'systemctl stop' call to finish before the agent completes shutdown, resulting in the agent process not exiting gracefully. [Support ticket: #37270] |
+| Bug Fix | Handle a case where an agent on a failed/rebooted standby node attempts to join the cluster before the previous agent from the same host has timed out. [Support ticket: #43162] |
+| Bug Fix | Removed 'efm upgrade-conf' support for very old Failover Manager version passwords. In rare cases, this support was causing upgrade failures. |
```

Comment:
I looked it up and "very old" means before version 4.0. If you think it would be better to call out the version here, that might look better than what I have.